fighters/common/src/general_statuses/mod.rs

```diff
@@ -141,7 +141,7 @@ fn nro_hook(info: &skyline::nro::NroInfo) {
             sub_transition_group_check_ground_jump_mini_attack,
             sub_transition_group_check_air_escape,
             sub_transition_group_check_ground_escape,
-            sub_transition_group_check_ground_guard
+            sub_transition_group_check_ground
```

Comment:
You removed the `sub_transition_group_check_ground_guard` install, was that intentional?

Reply:
> You removed the `sub_transition_group_check_ground_guard` install, was that intentional?

No it was not…

Reply:
lmao gottem